provisioning/roles/php-memcached/meta/main.yml

```diff
@@ -1,4 +1,5 @@
 ---
 dependencies:
+  - { role: php-fpm }
```

Comment:
Ah yes, you could add a `when` instruction to run different notifications depending on the PHP role used. Maybe you'll need to register something in both the `php-fpm` and `php-apache` role to know which one is used. Check the databases roles for an example (eg. https://github.com/liip/drifter/blob/master/provisioning/roles/postgresql/tasks/main.yml).